diff --git a/node_modules/@kentcdodds/tmp-react-server-dom-esm/cjs/react-server-dom-esm-client.browser.development.js b/node_modules/@kentcdodds/tmp-react-server-dom-esm/cjs/react-server-dom-esm-client.browser.development.js
index cbab9ac..6d5fe76 100644
--- a/node_modules/@kentcdodds/tmp-react-server-dom-esm/cjs/react-server-dom-esm-client.browser.development.js
+++ b/node_modules/@kentcdodds/tmp-react-server-dom-esm/cjs/react-server-dom-esm-client.browser.development.js
@@ -95,7 +95,8 @@ function preloadModule(metadata) {
     return existingPromise;
   } else {
     // $FlowFixMe[unsupported-syntax]
-    var modulePromise = import(metadata.specifier);
+    //var modulePromise = import(metadata.specifier);
+    const modulePromise = window.resolveRSCModule(metadata.specifier);
     modulePromise.then(function (value) {
       var fulfilledThenable = modulePromise;
       fulfilledThenable.status = 'fulfilled';
diff --git a/node_modules/@kentcdodds/tmp-react-server-dom-esm/cjs/react-server-dom-esm-client.browser.production.js b/node_modules/@kentcdodds/tmp-react-server-dom-esm/cjs/react-server-dom-esm-client.browser.production.js
index b366063..9cf2661 100644
--- a/node_modules/@kentcdodds/tmp-react-server-dom-esm/cjs/react-server-dom-esm-client.browser.production.js
+++ b/node_modules/@kentcdodds/tmp-react-server-dom-esm/cjs/react-server-dom-esm-client.browser.production.js
@@ -48,7 +48,8 @@ function preloadModule(metadata) {
     return existingPromise;
   } else {
     // $FlowFixMe[unsupported-syntax]
-    const modulePromise = import(metadata.specifier);
+    //const modulePromise = import(metadata.specifier);
+    const modulePromise = window.resolveRSCModule(metadata.specifier);
     modulePromise.then(value => {
       const fulfilledThenable = modulePromise;
       fulfilledThenable.status = 'fulfilled';
